docker/dev/Dockerfile

```diff
@@ -192,7 +202,16 @@ RUN $CMAKE --build $BASE_PATH/parflow/build
 
 USER root
 RUN $CMAKE --install $BASE_PATH/parflow/build --prefix /opt/parflow
+
+# Install the locally built version of pftools
 USER ubuntu
+RUN python3 -m pip install /home/ubuntu/parflow/build/pftools/python
+RUN cp -R /home/ubuntu/parflow/build/pftools/python/parflow/tools/ref ~/.local/lib/python3.8/site-packages/parflow/tools/ref
+
+RUN rm -R /home/ubuntu/parflow/src
```

Comment:
We remove the original source code after build to allow us to rebuild from the source on our machine by mounting it into this location that will now be empty

Reply:
There might be a better way to do it. You can still mount a local src over the docker one without removing anything.